web/geosearch/datapunt_geosearch/db.py

```diff
@@ -16,16 +16,17 @@ def retry_on_psycopg2_error(func):
     """
     @functools.wraps(func)
     def wrapper_retry(*args, **kwargs):
-        retry = 3
-        while retry > 0:
+        retry = 0
```

Comment:
Evt. zou het ook met for-loopje kunnen, dan heb je geen loop vars nodig:
``` 
for _ in range(3):
    try:
        print('try it')
    except Exeption:
        print('crash')
    else:
        print('it worked, leave loop')
        break
else:
    raise
```

Reply:
Kan ook met een recursie :D Maar het is wat het is, zo minder mogelijk aanrakken.